tests/python/pants_test/backend/jvm/tasks/jvm_compile/java/test_zinc_compile_integration.py

```diff
@@ -132,29 +134,72 @@ def test_no_zinc_file_manager(self):
         pants_run = self.run_test_compile(workdir, cachedir, target_spec, clean_all=True)
         self.assertEqual(0, pants_run.returncode)
 
-  def test_failed_hermetic_incremental_compile(self):
-    with temporary_dir() as cache_dir:
+  def test_hermetic_incremental_compile(self):
+    """
+    1) create a target containing two scala files
+    2) compile the target, which would be a full compile
+    3) modify a scala file slightly
+    4) recompile, and make sure the compile is incremental by checking the zinc outputs
+    """
+
+    def _create_file(path, value):
```

Comment:
`TestBase` has a `create_file` function - is there a reason not to use it?

Reply:
Good point. Adjusted.